src/aap_eda/api/metadata.py

```diff
@@ -13,7 +13,9 @@ def determine_actions(self, request, view):
         the fields that are accepted for 'PUT' and 'POST' methods.
         """
         actions = {}
-        for method in {"PUT", "PATCH", "POST"} & set(view.allowed_methods):
+        for method in {"GET", "PUT", "PATCH", "POST"} & set(
```

Comment:
> Why GET is not on the list in the first place?

Because [DRF didn't put it in](https://github.com/encode/django-rest-framework/blob/fe92f0dd0d4c587eed000c7de611ddbff241bd6a/rest_framework/metadata.py#L78). However, [AWX added GET back in](https://github.com/ansible/awx/blob/d94f766fcbe5cf68370830430d980b67d6cfd164/awx/api/metadata.py#L192).

Why did DRF not put it in? This is speculative, but I suspect it is considered self-obvious. Permissions for OPTIONS requests almost always are the same as a GET request, so if you received the OPTIONS response _at all_ then you know that you can perform the GET option.

So back up, why this issue? I believe this issue isn't about knowing whether GET is allowed or not, but about the specific fields/content from a GET request.